test/test_clustohttp.py

```diff
@@ -0,0 +1,202 @@
+import clustohttp
```

Comment:
:nit these should be organized according to PEP8: system/standard imports first, 3rd party, and then local